Cargo.toml

```diff
@@ -3,20 +3,22 @@ name = "human_name"
 version = "1.0.1"
 authors = ["David Judd <david.a.judd@gmail.com>"]
 description = "A library for parsing and comparing human names"
+keywords = ["human", "name", "language", "nlp"]
+categories = ["parsing", "text-processing", "command-line-utilities",]
 license = "Apache-2.0"
 repository = "https://github.com/djudd/human-name"
-documentation = "http://djudd.github.io/human-name"
+documentation = "https://docs.rs/human_name/"
```

Comment:
I'd forgotten about this GitHub site completely, but when I set it up, I'd also set up `.travis.yml` to automatically update it. I have no idea if that's still working. Would you mind also removing the relevant likes from `.travis.yml`? 

https://github.com/djudd/human-name/blob/master/.travis.yml#L9-L17

Reply:
Done in https://github.com/djudd/human-name/pull/7/commits/6afab67009c8ce55bc068b91223d17897fe85eab.
Perhaps move to GH Actions in another PR?